src/webapp/WEB-INF/templates/home.handlebars

```diff
@@ -4,40 +4,63 @@
     {{translate 'technical_setup_instruction'}}
 </div>
 <div id="feedback-links">
-    <div class="feedback-box" id="feedback-content-problem">
-        <h3>{{translate 'content_title'}}</h3>
-        <ul>{{translate 'content_explanation'}}</ul>
-        <p><a id="feedback-report-content-link" href="javascript:;">{{translate 'content_link'}}</a></p>
-        <div class="clear"></div>
-    </div>
 
-    <div class="feedback-box" id="feedback-ask-help">
-        <h3>{{translate 'ask_title'}}</h3>
-        <ul>{{translate 'ask_explanation'}}</ul>
-        <p><a id="feedback-report-helpdesk-link" href="javascript:;">{{translate 'technical_link'}}</a></p>
-        <div class="clear"></div>
-    </div>
+    {{#if showContentPanel}}
+        <div class="feedback-box" id="feedback-content-problem">
+            <h3>{{translate 'content_title'}}</h3>
+            <ul>{{translate 'content_explanation'}}</ul>
+            <p><a id="feedback-report-content-link" href="javascript:;">{{translate 'content_link'}}</a></p>
+            <div class="clear"></div>
+        </div>
+    {{/if}}
 
-    <div class="feedback-box" id="feedback-tech-problem">
-        <h3>{{translate 'technical_title'}}</h3>
-        <ul>{{translate 'technical_explanation'}}</ul>
-        <p><a id="feedback-report-technical-link" href="javascript:;">{{translate 'technical_link'}}</a></p>
-        <div class="clear"></div>
-    </div>
+    {{#if showHelpPanel}}
+        <div class="feedback-box" id="feedback-ask-help">
+            <h3>{{translate 'ask_title'}}</h3>
+            <ul>{{translate 'ask_explanation'}}</ul>
+            <p><a id="feedback-report-helpdesk-link" href="javascript:;">{{translate 'technical_link'}}</a></p>
+            <div class="clear"></div>
+        </div>
+    {{/if}}
 
-    <div class="feedback-box" id="feedback-improvement">
-        <h3>{{translate 'suggest_title'}}</h3>
-        <ul>{{translate 'suggest_explanation'}}</ul>
-        <p><a id="feedback-suggest-feature-link" href="{{featureSuggestionUrl}}" target="_blank">{{translate 'suggest_link'}}</a></p>
-        <div class="clear"></div>
-    </div>
+    {{#if showTechnicalPanel}}
+        <div class="feedback-box" id="feedback-tech-problem">
+            <h3>{{translate 'technical_title'}}</h3>
+            <ul>{{translate 'technical_explanation'}}</ul>
+            <p><a id="feedback-report-technical-link" href="javascript:;">{{translate 'technical_link'}}</a></p>
+            <div class="clear"></div>
+        </div>
+    {{/if}}
+
+    {{#if showSuggestionsPanel}}
+        <div class="feedback-box" id="feedback-improvement">
+            <h3>{{translate 'suggest_title'}}</h3>
+            <ul>{{translate 'suggest_explanation'}}</ul>
+            <p><a id="feedback-suggest-feature-link" href="{{featureSuggestionUrl}}" target="_blank">{{translate 'suggest_link'}}</a></p>
+            <div class="clear"></div>
+        </div>
+    {{/if}}
+
+    {{#unless showContentPanel}}
```

Comment:
Don't need to fix this, but you're going against mustache here. The idea is that all you're logic should be in the code and the template should be simple, that's why there aren't any boolean operators in the language. The nested `unless` are just a hack.